src/components/action-button.js

```diff
@@ -19,7 +19,7 @@ export default function ActionButton({ layout, button, Theme, engineApp, context
       const { actions } = layout;
       actions.forEach(action => {
         const findAction = actionsList.find(act => act.value === action.actionType);
-        actionList.push(findAction.promise({ engineApp, ...action }));
+        findAction && actionList.push(findAction.call({ engineApp, ...action }));
```

Comment:
Let's add check on call here as well:

typeof findAction.call === 'function'  or just findAction.call

Reply:
sure, on it